queues.json

```diff
@@ -0,0 +1 @@
+{"dummy": "tii2q", "iqm5q_q0": "iqm5q", "iqm5q": "iqm5q", "qw5q_gold_qblox": "qw5q_gold", "tii1q_b1": "tii1q_b1"}
```

Comment:
This is also not needed here as we should use the one from the other repository. Otherwise we have duplication and we need to keep both updated. Also qibolab should be "lab agnostic", everything TII specific should go in qibolab_platforms_qrc.
In the workflow you can still read this using 
```sh
queues=`cat qibolab_platforms_qrc/queues.json`
``` 
after cloning the other repo.

Reply:
Done.